### ISSUE TITLE
Calling the Hide method does not work

### ISSUE THREAD

**Issue by AlmirKadric** [bitpay/cordova-plugin-qrscanner#181]
Currently when the `Hide()` method is called, nothing happens and the camera is still visible. However if I call the `Destroy()` the camera is stopped however the background is set to a blueish gray color.

**Comment by yangnana11** [comment]
Hello,

It's still dark blue or black, not return to it's earlier stage...

**Comment by thekingrenz23** [comment]
@yangnana11 i followed your instruction about the blue color when the qrscanner is destroyed but it didt solved it

**Comment by thekingrenz23** [comment]
> Hello,
> 
> For destroy function, it's still dark blue or black color for latest version (2.6.0)
> There is a workaround for hide function which worked fine. You should change the color to WHITE instead of TRANSPARENT in java file. I tested with Android, I haven't tested with IOS though.
> 
> ```
>     private void makeOpaque() {
>         this.cordova.getActivity().runOnUiThread(new Runnable() {
>             @Override
>             public void run() {
>                 webView.getView().setBackgroundColor(Color.WHITE);
>             }
>         });
>         showing = false;
>     }
> ```

i follwed this on my android project didnt solved it

**Comment by AkaashMukherjee** [comment]
Yeah I have the same error, hide doesn't work and destroy sets to a dark color

**Comment by p3v9d5ui** [comment]
Any more information on this?

**Comment by p3v9d5ui** [comment]
I have created a pull request that fixes this (though I actually didn't experience a blue background; instead it remained transparent, but maybe still the same issue):

#279

This bug was created by this previous commit:

c9531b8

Or, if my PR is not accepted, you can use my clone:

https://github.com/p3v9d5ui/cordova-plugin-qrscanner

**Comment by gerarts** [comment]
Any reason #279 hasn't been merged yet?
